File: javascript-closure-angular/API/Client/ResponseRoute.js
goog.provide('API.Client.ResponseRoute');

/**
 * @record
 */
API.Client.ResponseRoute = function() {}

/**
 * @type {!Date}
 * @export
 */
API.Client.ResponseRoute.prototype.departureTime;

/**
 * @type {!Date}
 * @export
 */
API.Client.ResponseRoute.prototype.arrivalTime;

/**
 * @type {!Array<!API.Client.ResponseRoutePart>}
 * @export
 */
API.Client.ResponseRoute.prototype.parts;

File: scala-play-server/app/model/ResponseRoute.scala
package model

import play.api.libs.json._
import java.time.OffsetDateTime

/**
  * Represents the Swagger definition for ResponseRoute.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseRoute(
  departureTime: OffsetDateTime,
  arrivalTime: OffsetDateTime,
  parts: List[ResponseRoutePart]
)

object ResponseRoute {
  implicit lazy val responseRouteJsonFormat: Format[ResponseRoute] = Json.format[ResponseRoute]
}

